game/scenes.py

```diff
@@ -694,24 +694,26 @@ def generate_world(self):
                         self.add_block((x, y + dy, z), BEDSTONE, immediate=False)
 
         # generate the hills randomly
-        o = n - 10
-        for _ in range(120):
-            a = random.randint(-o, o)  # x position of the hill
-            b = random.randint(-o, o)  # z position of the hill
-            c = -1  # base of the hill
-            h = random.randint(1, 6)  # height of the hill
-            s = random.randint(4, 8)  # 2 * s is the side length of the hill
-            d = 1  # how quickly to taper off the hills
-            block = random.choice([DIRT_WITH_GRASS, SNOW, SAND])
-            for y in range(c, c + h):
-                for x in range(a - s, a + s + 1):
-                    for z in range(b - s, b + s + 1):
-                        if (x - a) ** 2 + (z - b) ** 2 > (s + 1) ** 2:
-                            continue
-                        if (x - 0) ** 2 + (z - 0) ** 2 < 5 ** 2:  # 6 = flat map
-                            continue
-                        self.add_block((x, y, z), block, immediate=False)
-                s -= d  # decrement side lenth so hills taper off
+
+        if HILLSON is True:
```

Comment:
To avoid unnecessary white space, it would be better to return on the False condition. For example: 
```
if not HILLSON:
    return
o = n - 10
...
```

Reply:
Will fix this. Thanks.